experimental/framework/tinylicious-client/src/TinyliciousClient.ts

```diff
@@ -117,46 +117,50 @@ export class TinyliciousClientInstance {
 }
 
 /**
- * Singular global instance that lets the developer define all Container interactions with the Tinylicious service
+ * TinyliciousClient static class with singular global instance that lets the developer define
+ * all Container interactions with the Tinylicious service
  */
-let globalTinyliciousClient: TinyliciousClientInstance | undefined;
-export const TinyliciousClient = {
-    init(serviceConnectionConfig?: TinyliciousConnectionConfig) {
-        if (globalTinyliciousClient) {
+// eslint-disable-next-line @typescript-eslint/no-extraneous-class
+export class TinyliciousClient {
+    private static globalTinyliciousClient: TinyliciousClientInstance | undefined;
+    static init(serviceConnectionConfig?: TinyliciousConnectionConfig) {
+        if (this.globalTinyliciousClient) {
```

Comment:
For static members, should refer to them like `TinyliciousClient.globalTinyliciousClient` rather than using `this`.

Reply:
Oh even within the class itself? I'll update it!

Reply:
Updated to be `TinyliciousClient.globalInstance` to avoid repeating TinyliciousClient in the naming